sno/working_copy/gpkg.py

```diff
@@ -164,14 +138,13 @@ def is_initialised(self):
         if not self.is_created():
             return False
         with self.session() as db:
-            dbcur = db.cursor()
-            dbcur.execute(
+            r = db.execute(
```

Comment:
I always get confused between sessions and connections and engines..

Check this `db` thing - does it have a `scalar` method?

```python
return db.scalar(sql, params)
```
rather tthan .execute()/.fetchone


If not, at a minimum you can call `.scalar()` on the resultset, which just looks a bit tidier than `.fetchone()[0]`

Reply:
Did a bit of a search/replace for fetchone()[0] in this PR but will tidy it up properly in a follow up PR.

Reply:
Maybe we should use `(sa_)?{engine,conn,session}` names rather than `db`?

Reply:
Will rename db in a follow up PR - maybe once I have postGIS using sqlalchemy too.

Right now, some of these db's are  "either a sqlalchemy-session, or a psypopg2 cursor" which actually have similar-enough functionality that some things work on both. So, cleaning up naming seems easiest once we have sqlalchemy everywhere and it's clear at least what everything is